frontend/src/components/Tree/Tree.jsx

```diff
@@ -178,8 +181,8 @@ export function Tree({ jsonData, onNodeClick }) {
       entries.forEach((entry) => {
         setDimensions({
           // add margins to prevent chopped off content
```

Comment:
Remove this comment since it doesn't seem to apply anymore?

Reply:
Haha my bad. Thanks for catching that.